dotnet/samples/Concepts/ChatCompletion/Connectors_WithMultipleLLMs.cs

```diff
@@ -26,57 +27,92 @@ public async Task RunAsync()
                 serviceId: "OpenAIChat")
             .Build();
 
-        await RunByServiceIdAsync(kernel, "AzureOpenAIChat");
-        await RunByModelIdAsync(kernel, TestConfiguration.OpenAI.ChatModelId);
-        await RunByFirstModelIdAsync(kernel, "gpt-4-1106-preview", TestConfiguration.AzureOpenAI.ChatModelId, TestConfiguration.OpenAI.ChatModelId);
+        // Preconfigured function settings
+        await PreconfiguredFunctionSettingsByFirstModelIdAsync(kernel, ["gpt-4-1106-preview", TestConfiguration.AzureOpenAI.ChatModelId, TestConfiguration.OpenAI.ChatModelId]);
+        await PreconfiguredFunctionSettingsByFirstServiceIdAsync(kernel, ["NotFound", "AzureOpenAIChat", "OpenAIChat"]);
+        await PreconfiguredFunctionSettingsByModelIdAsync(kernel, TestConfiguration.OpenAI.ChatModelId);
+        await PreconfiguredFunctionSettingsByServiceIdAsync(kernel, "AzureOpenAIChat");
+
+        // Per invocation settings
+        await InvocationSettingsByServiceIdAsync(kernel, "AzureOpenAIChat");
```

Comment:
I think `InvokePromptByServiceIdAsync` would be clearer. Again a comment for each of these functions explaining what it does.